src/hotspot/os/windows/os_windows.cpp

```diff
@@ -4211,7 +4168,13 @@ void os::win32::initialize_system_info() {
   DWORD processors = 0;
   bool schedules_all_processor_groups = win32::is_windows_11_or_greater() || win32::is_windows_server_2022_or_greater();
   if (schedules_all_processor_groups) {
-    processors = system_logical_processor_count();
+    processors = GetActiveProcessorCount(ALL_PROCESSOR_GROUPS);
```

Comment:
This doesn't seem right. Don't we want to know the number of processors available to the JVM not the number in the entire "system"?